examples/stable_diffusion_v2/tests/st/_common.py

```diff
@@ -35,5 +35,24 @@ def gen_dummpy_data():
     return data_dir
 
 
+def down_checkpoint(version="1.5"):
+    __dir__ = os.path.dirname(os.path.abspath(__file__))
```

Comment:
`__dir__` is a internal method for python class, better to have a better name, like `root`

Reply:
fixed